cicd/tekton/tekton-main-app.yaml

```diff
@@ -6,12 +6,12 @@ metadata:
 spec:
   project: default
   source:
-    repoURL: https://github.com/intelops/capten-templates.git
-    targetRevision: HEAD
+    repoURL: https://github.com/intelops/capten-templates
+    targetRevision: main
     path: cicd/tekton/argocd-apps
     helm:
       valueFiles:
       - values.yaml
   destination:
     server: https://kubernetes.default.svc
-    namespace: tekton-pipelines
+    namespace: argo-cd
```

Comment:
why argo-cd namespace ?

Reply:
this is a instance of argo-cd application so added there